packages/react-icons/README.md

```diff
@@ -0,0 +1,11 @@
+# @fluentui/react-icons
+
+**ReactIcons components for [Office UI Fabric React](https://dev.microsoft.com/fabric)**
```

Comment:
Can you update the content of this file? This line should be as follows but you should also look over the rest of the content.
```suggestion
**Icon components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
```